ui/src/components/common/NotificationCenter.scss

```diff
@@ -0,0 +1,28 @@
+@mixin notification-center {
+  .return-link {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    margin-bottom: 20px;
```

Comment:
Is there a `spacing` variable we can use here instead of hard coded px?

Reply:
Sure. I'll check which one to use :)